src/main/java/cz/muni/fi/pv286/mmt/exceptions/InvalidArrayValueException.java

```diff
@@ -0,0 +1,17 @@
+package cz.muni.fi.pv286.mmt.exceptions;
+
+import java.io.IOException;
+
+/**
+ * An exception that is thrown when an invalid data format is found.
+ */
+public class InvalidArrayValueException extends IOException {
+    public InvalidArrayValueException(String message) {
+        super(message);
+    }
```

Comment:
**suggestion**: Remove unused function.

It's a code smell and the code it's harder to read.